smsdk/client_v0.py

```diff
@@ -318,6 +318,7 @@ def inner(
 
             if not "_limit" in kwargs:
                 print("_limit not specified.  Maximum of 5000 rows will be returned.")
+                kwargs["_limit"] = 5000
```

Comment:
Let's add in some minor protecting the users from themselves and make it so if they set a limit over 5000 we print a warning saying "Warning: _limit over 5000 specified.  Setting too large of limits can lead to timeouts,"

Reply:
Done.